This is exception handling in python